docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[[ml-dfa-limitations]]
+== {dfanalytics-cap} limitations
+
+experimental[]
+
+The following limitations and known problems apply to the 7.3 release of 
+the Elastic {dfanalytics} feature:
+
+[float]
+[[dfa-ccs-limitations]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.3 for {dfanalytics}.
+
+[float]
+[[dfa-deletion-limitations]]
+=== Deleting a {dfanalytics-job} does not delete the {dataframe} destination index
+
+When deleting a {dfanalytics-job} using 
+`DELETE _ml/data_frame/analytics/<data_frame_analytics_id>` the {dataframe} 
+destination index won't be deleted. This object must be deleted separately.
+
+[float]
+[[dfa-update-limitations]]
+=== Cannot update a {dfanalytics-job}
+
+{dfanalytics-cap} configurations cannot be updated. Please delete and 
+then create a new {dfanalytics-job} instead.
+
+[float]
+[[dfa-datatype-limitations]]
+=== {dfanalytics-cap} data type limitation
+
+{dfanalytics-cap} only support numeric fields and don't support fields that 
+contain arrays. The analysis will skip every document that has such fields.
```

Comment:
It would be good to clarify here: The analysis will skip every document that has missing values of unsupported values for one or more analyzed fields.

Reply:
Also there is overlap between this section and the last two. I wonder if the last two are enough to communicate what this one is trying to.

Reply:
@dimitris-athanasiou I deleted this section and merged its content into the "Field limitations in data frame analytics" subsection together with the categorial field limitations. I left the limitation of the missing values unchanged.